torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3167,6 +3167,51 @@ def sample_inputs_take_along_dim(op_info, device, dtype, requires_grad, **kwargs
             )
 
 
+def error_inputs_amax_amin(op_info, device, **kwargs):
+
+    # Error Inputs for zero-dim tensors, when 'dim' arg is not provided.
+    shape = (S, 0, S)
+    err_msg1 = "Specify the reduction dim with the 'dim' argument."
+    yield ErrorInput(SampleInput(torch.rand(shape, device=device)), error_type=RuntimeError,
+                     error_regex=err_msg1)
+
+    # Error Inputs for tensors with more than 64 dimension
+    sizes = [1] * 65
+    err_msg2 = "only tensors with up to 64 dims are supported"
+    yield ErrorInput(SampleInput(torch.randn(sizes, device=device), kwargs={'dim': -1}),
+                     error_type=RuntimeError, error_regex=err_msg2)
+    yield ErrorInput(SampleInput(torch.randn(sizes, device=device), kwargs={'dim': 64}),
+                     error_type=RuntimeError, error_regex=err_msg2)
+
+    # Error Inputs for mixed devices
```

Comment:
mixed devices are already tested in regular `test_out`, no need for additional cases here.

Reply:
Sure, I've removed it. Thanks